subprojects/core/src/main/java/org/gradle/api/internal/tasks/DefaultTaskContainer.java

```diff
@@ -603,9 +543,6 @@ public void setModelNode(MutableModelNode modelNode) {
     @Override
     public void whenElementKnown(Action<? super ElementInfo<Task>> action) {
         super.whenElementKnown(action);
-        for (Map.Entry<String, TaskProvider<?>> entry : placeholders.entrySet()) {
```

Comment:
isn't this an empty method now that just calls `super.<same method>(<same args>)`?